packages/pn-pa-webapp/src/pages/ApiKeys.page.tsx

```diff
@@ -74,26 +74,22 @@ const ApiKeys = () => {
 
   const apiKeyBlocked = (apiKeyId: string) => {
     handleCloseModal();
-    // Integrare logica di success / failure e eventuale callback relativa (aggiornamento tabella per esempio)
-    void dispatch(setApiKeyStatus({ apiKey: apiKeyId, status: ApiKeyStatus.BLOCKED }));
+    void dispatch(setApiKeyStatus({ apiKey: apiKeyId, status: ApiKeySetStatus.BLOCK })).then(() => void dispatch(getApiKeys()));
   };
 
   const apiKeyEnabled = (apiKeyId: string) => {
     handleCloseModal();
-    // Integrare logica di success / failure e eventuale callback relativa (aggiornamento tabella per esempio)
-    void dispatch(setApiKeyStatus({ apiKey: apiKeyId, status: ApiKeyStatus.ENABLED }));
+    void dispatch(setApiKeyStatus({ apiKey: apiKeyId, status: ApiKeySetStatus.ENABLE })).then(() => void dispatch(getApiKeys()));
   };
 
   const apiKeyRotated = (apiKeyId: string) => {
     handleCloseModal();
-    // Integrare logica di success / failure e eventuale callback relativa (aggiornamento tabella per esempio)
-    void dispatch(setApiKeyStatus({ apiKey: apiKeyId, status: ApiKeyStatus.ROTATED }));
+    void dispatch(setApiKeyStatus({ apiKey: apiKeyId, status: ApiKeySetStatus.ROTATE })).then(() => void dispatch(getApiKeys()));
   };
 
   const apiKeyDeleted = (apiKeyId: string) => {
     handleCloseModal();
-    // Integrare logica di success / failure e eventuale callback relativa (aggiornamento tabella per esempio)
-    void dispatch(setApiKeyDeleted(apiKeyId));
+    void dispatch(deleteApiKey(apiKeyId)).then(() => void dispatch(getApiKeys()));
   };
 
   return (
```

Comment:
The eventual situation of an error when the data is fetched through the API should be considered. Exactly for this, I created the `ApiErrorWrapper` component, and for less straightforward cases, the `ApiError` component and the `useErrors` hook. Cfr. https://pagopa.atlassian.net/wiki/spaces/PN/pages/583173517/Gestione+degli+errori+nelle+chiamate+API

Reply:
I suggesst you to use `ApiErrorWrapper` inside the outermost `Box`, and below the title. In that way, the title will be shown even in the case of an API error.
![image](https://user-images.githubusercontent.com/5519535/205610711-22104cd7-020f-4621-9d2c-50b902226ba4.png)

With the current code in`ApiKeys.page.tsx`, only the "API call failed" is shown
![image](https://user-images.githubusercontent.com/5519535/205610608-138f5240-cb3e-4981-b300-c55fd8a6cab8.png)